source/openapi-admin-v3.yaml

```diff
@@ -2360,6 +2360,112 @@ paths:
           description: Deleted
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/security/private_endpoints":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+    get:
+      operationId: adminGetPrivateEndpoints
+      tags:
+        - security
+      summary: List VPC Private Endpoints
+      description: |
+        Get a list of all private endpoints configured for the App.
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PrivateEndpoint"
+        404:
+          description: Not Found
+    post:
+      operationId: adminCreatePrivateEndpoints
+      tags:
+        - security
+      summary: Create a VPC Private Endpoint
+      description: |
+        Configure a new private endpoint for the App.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PrivateEndpoint"
+      responses:
+        201:
+          description: Created
+        400:
+          description: Bad Request
+  "/groups/{groupId}/apps/{appId}/security/private_endpoints/{privateEndpointId}":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/PrivateEndpointId"
+    get:
+      operationId: adminGetPrivateEndpoint
+      tags:
+        - security
+      summary: Get a VPC Private Endpoint
+      description: |
+        Get a specific private endpoint configuration.
+      responses:
+        200:
+          description: Found
```

Comment:
The description says "Get a specific private endpoint configuration" but the response is just 200/Found - do we have an example we could add, or does it not return anything aside from the code?